pkgs/misc/tmux-plugins/tmux-thumbs/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, mkTmuxPlugin, fetchFromGitHub, thumbs }:
+mkTmuxPlugin rec {
+  pluginName = "tmux-thumbs";
+  version = "0.7.1";
+  rtpFilePath = "tmux-thumbs.tmux";
+  src = fetchFromGitHub {
+    owner = "fcsonline";
+    repo = pluginName;
+    rev = version;
+    sha256 = "sha256-PH1nscmVhxJFupS7dlbOb+qEwG/Pa/2P6XFIbR/cfaQ=";
+  };
+  patches = [ ./fix.patch ];
+  postInstall = ''
+    substituteInPlace $target/tmux-thumbs.sh --replace '@@tmux-thumbs-binary@@' '${thumbs}/bin/tmux-thumbs'
+  '';
```

Comment:
Please combine this like https://github.com/NixOS/nixpkgs/blob/51ceca24be1958d2ab9385b3d56162658a5c98cd/pkgs/applications/audio/curseradio/default.nix#L20-L25

Reply:
I am not sure if it is possible, at last I haven't found a way to do this. The substitution uses `out` but I cannot use it as a nix variable because the derivation hasn't been created yet. 

Following code correctly complains about `out` being undefined:
```
 patches = [
    (substituteAll {
      src = ./fix.patch;
      thumbsBinDir = "${out}/bin";
    })
  ];
```

Am I missing something?